exec/java-exec/src/main/java/org/apache/drill/exec/util/FileSystemUtil.java

```diff
@@ -42,6 +47,12 @@ public class FileSystemUtil {
 
   private static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(FileSystemUtil.class);
 
+  private static int recursiveListingMaxSize;
+
+  static {
+    recursiveListingMaxSize = DrillConfig.create().getInt(ExecConstants.RECURSIVE_FILE_LISTING_MAX_SIZE);
+  }
```

Comment:
This route to the config option felt pretty weird, I don't if there's a better way?

Reply:
This looks wonky but correct.

Reply:
That it might be a heavyweight duplication of work was bothering me enough that I went and timed it. It takes about ~100ms when I start embedded Drill locally. That's just enough to make me wonder if it's worth trying to redesign this stuff so that it loads from an existing instance of DrillConfig instead of constructing its own.

Reply:
It's gone now.